bin/machinery

```diff
@@ -20,6 +20,12 @@
 require_relative '../lib/machinery'
 
 begin
+  if Dir.exist?(File.join(Machinery::ROOT, ".git"))
+    Dir.chdir(File.join(Machinery::ROOT, "machinery-helper")) do
+      Cheetah.run("rake")
```

Comment:
During the discussion yesterday we said that it would be cleaner to extract that login from the Rakefile into a separate class that could be used from both the Rakefile and machinery itself. Can we still do it like that?